components/SignInForm.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import { Field, reduxForm } from 'redux-form';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import shortid from 'shortid';
+
+const renderField = ({ input, label, type, meta: { touched, error } }) =>
+  <div>
+    <input {...input} placeholder={label} type={type} />
+    {touched &&
+      error &&
+      <div>
+        {error}
+      </div>}
+  </div>;
+
+const renderErrors = errors =>
```

Comment:
It could be good if it's been different component so we can use it on every error message.